docs/Contributing/architecture/software/software-policies.md

```diff
@@ -0,0 +1,130 @@
+# Software Policies Architecture
```

Comment:
This needs to be deleted. We mean something completely different when we say "Policies" and don't actually have the functionality listed here implemented.

Reply:
Emptying and leaving headers.